omnibus/config/software/inspec.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# All Rights Reserved.
+#
+
+name 'inspec'
+
+dependency 'ruby'
+dependency 'train'
+
+license "Apache-2.0"
+license_file "LICENSE"
+
+default_version "v0.18.0"
```

Comment:
I thinks this is addressed here: https://github.com/chef/omnibus-software/blob/master/config/software/chef.rb#L22-L35